Description: Remove Switch module because now it's in perl core
Origin: vendor
Forwarded: not-needed
Author: Alejandro Garrido Mota <garridomota@debian.org>
Last-Update: 2012-05-01

--- a/gbackground
+++ b/gbackground
@@ -4,7 +4,6 @@
 use Gtk2::GladeXML;
 use UI::Dialog;
 use UI::Dialog::Backend::Zenity;
-use Switch;
 use YAML;
 
 ## Variables Declaration
